spec/index.html

```diff
@@ -925,8 +952,6 @@ <h3>Graph Comparison</h3>
       the triple (|s|, |p|, |o|) is in |G| if and only if
       the triple ( |M|(|s|), |M|(|p|), |M|(|o|) ) is in <var>G'</var>.</p>
 
-    <p>See also: <a>IRI equality</a>, <a>literal term equality</a>.</p>
```

Comment:
Note that I have removed this one because the cross-references to these definitions are integrated directly into the definition above now.